accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -204,8 +619,11 @@ Windows-specific:
     - `AddMulticastGroupOnInterface`
     - `DeleteMulticastGroupFromInterface`
 * **Socket**
+    - `.ctor(SocketInformation)`
+    - `.ctor(SafeSocketHandle)`
```

Comment:
Is this actually Windows-specific?

Reply:
@dotnet/ncl

Reply:
`SocketInformation` variant is; `SafeSocketHandle` variant is not.

Reply:
Removed `.ctor(SafeSocketHandle)`